l1-contracts/contracts/common/libraries/DynamicIncrementalMerkle.sol

```diff
@@ -59,6 +59,24 @@ library DynamicIncrementalMerkle {
         return bytes32(0);
```

Comment:
Still, you should fix the comment for the `setup` function, which says that calling it resets the tree